tests/contracts/integration/test_intercontract_integration.py

```diff
@@ -1192,54 +1201,6 @@ def test_upgrading_adjudicator(testerchain,
     assert adjudicator_v1 == adjudicator.functions.target().call()
 
 
-def test_another_slashing(testerchain,
```

Comment:
Why was this removed?

Reply:
Really hard to maintain (in this PR hard to make it alive), also we have this test in contracts unit tests.
But if it's an issue - let's open one

Reply:
Dunno, @cygnusv what are your thoughts?